duckduckgo/iOS
# Implement daily pixels for fire button

Task/Issue URL: https://app.asana.com/0/414235014887631/1204198140064640/f

**Description**:
After discussion how to properly measure feature usage we identified a need for "daily pixels" - pixels that are limited to be sent at most once per-day per-user.
This PR also includes two daily pixels for fire button tap and actual data clearing.

**Steps to test this PR**:
1. Check if the new fire button tap and data clear pixels are sent only once per day.
2. Review DailyPixel implementation and tests.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
